edx-platform/pearson-vue-theme/lms/templates/header/user_dropdown.html

```diff
@@ -43,17 +42,23 @@
         % if resume_block:
             <div class="mobile-nav-item dropdown-item dropdown-nav-item"><a href="${resume_block}" role="menuitem">${_("Resume your last course")}</a></div>
         % endif
-        % if not enterprise_customer_portal:
-            <div class="mobile-nav-item dropdown-item dropdown-nav-item"><a href="${reverse('dashboard')}" role="menuitem">${_("Dashboard")}</a></div>
-        % else:
-            <div class="mobile-nav-item dropdown-item dropdown-nav-item"><a href="${settings.ENTERPRISE_LEARNER_PORTAL_BASE_URL}/${enterprise_customer_portal.get('slug')}" role="menuitem">${_("Dashboard")}</a></div>
+        % if configuration_helpers.get_value('SHOW_DASHBOARD_IN_DROPDOWN', False):
+            <div class="mobile-nav-item dropdown-item dropdown-nav-item">
+                <a href="${reverse('dashboard')}" role="menuitem">${_("Dashboard")}</a>
+            </div>
+        % endif
+        % if configuration_helpers.get_value('SHOW_PROFILE_IN_DROPDOWN', False):
```

Comment:
Are we using this setting in any of the prod sites? @sergivalero20

Reply:
@Squirrel18 yes: 

![image](https://github.com/Pearson-Advance/openedx-themes/assets/73655023/ad07713d-6327-49bf-8821-2e166d3b4f24)